nmdc_runtime/api/core/auth.py

```diff
@@ -99,17 +111,21 @@ async def __call__(self, request: Request) -> Optional[str]:
         scheme, param = get_authorization_scheme_param(authorization)
         if not authorization or scheme.lower() != "bearer":
             if self.auto_error:
+                print("uh oh")
```

Comment:
Marking this as something someone may want to revisit (i.e. elaborate on, remove) as part of the upcoming "simplify auth" work.